### ISSUE TITLE
CATTLE_URL is not set

### ISSUE THREAD

**Issue by arvalaan** [chrisurwin/rancher-rebalancer#2]
When launching the docker container on Rancher 1.6.15, it stops immediately with an error saying `CATTLE_URL is not set`

**Comment by arvalaan** [comment]
Might just want to add it to your readme ;)

**Comment by chrisurwin** [comment]
How are you deploying this? Are you deploying from my catalog or just as a stand alone container?
If you are doing it stand alone then you need to set the label : ` io.rancher.container.agent.role: environment`

**Comment by arvalaan** [comment]
Yeah found your catalog a short while after. Which is amazing by the way. Maybe you should add that to the readme instead! It has some awesome resources.

**Comment by dannykopping** [comment]
@chrisurwin what's the name of your catalog?

**Comment by dannykopping** [comment]
nvm, found it: https://github.com/chrisurwin/catalog
